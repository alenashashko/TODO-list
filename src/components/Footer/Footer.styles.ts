import { makeStyles, createStyles, Theme } from "@material-ui/core";

export const useStyles = makeStyles((theme: Theme) =>
  createStyles({
    footer: {
      backgroundColor: theme.palette.primary.main,
      color: theme.palette.getContrastText(theme.palette.primary.main),
    },
    copyright: {
      textTransform: "uppercase",
      textAlign: "right",
    },
  })
);
