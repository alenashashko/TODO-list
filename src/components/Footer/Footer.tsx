import { FC } from "react";
import { Toolbar, Typography } from "@material-ui/core";

import { useStyles } from "./Footer.styles";

export const Footer: FC = () => {
  const classes = useStyles();

  return (
    <Toolbar className={classes.footer} component="footer">
      <Typography className={classes.copyright} variant="h6" component="h2">
        by Alena Shashko
      </Typography>
    </Toolbar>
  );
};
