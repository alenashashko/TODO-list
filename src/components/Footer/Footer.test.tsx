import { render, cleanup } from "@testing-library/react";
import { Footer } from "src/components/Footer";

afterEach(cleanup);

it("renders footer", () => {
  const { asFragment } = render(<Footer />);

  expect(asFragment()).toMatchSnapshot();
});
