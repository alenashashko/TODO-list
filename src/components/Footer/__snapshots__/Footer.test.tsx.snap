// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders footer 1`] = `
<DocumentFragment>
  <footer
    class="MuiToolbar-root MuiToolbar-regular makeStyles-footer-1 MuiToolbar-gutters"
  >
    <h2
      class="MuiTypography-root makeStyles-copyright-2 MuiTypography-h6"
    >
      by Alena Shashko
    </h2>
  </footer>
</DocumentFragment>
`;
